### DISCUSSION TITLE
LimeSDR-Mini  does not show up in device list

### DISCUSSION THREAD

**Discussion by fcapozzi62** [AlexandreRouma/SDRPlusPlus#515] [General]
I installed the V1.0.4 SDR#++ version and i connected the LimeSDR-mini device.  No serial number displayed in the device box. 
The are any configuration missing ??
Fabio.

**Comment by fcapozzi62** [comment]
... Platform is Windows 10.

**Reply by AlexandreRouma** [reply]
Make sure the LimeSDR drivers are installed. Follow the manufacturer's instructions for that.
